firmware_mod/scripts/mqtt-control.sh

```diff
@@ -240,6 +240,13 @@ done
     "${TOPIC}/motion/mqtt_snapshot")
       /system/sdcard/bin/mosquitto_pub.bin -h "$HOST" -p "$PORT" -u "$USER" -P "$PASS" -t "${TOPIC}"/motion/mqtt_snapshot ${MOSQUITTOPUBOPTS} ${MOSQUITTOOPTS} -m "$(motion_mqtt_snapshot status)"
     ;;
+        
+    "${TOPIC}/motion/mqtt_snapshot GET")
+      /system/sdcard/bin/getimage > "/tmp/mqtt_snapshot"
+      /system/sdcard/bin/jpegoptim -S 25k "/tmp/mqtt_snapshot"
+      /system/sdcard/bin/mosquitto_pub.bin -h "$HOST" -p "$PORT" -u "$USER" -P "$PASS" -t "${TOPIC}"/motion/snapshot/image ${MOSQUITTOPUBOPTS} ${MOSQUITTOOPTS} -f "/tmp/mqtt_snapshot"
```

Comment:
This can potentially lead to mixups for users that expect only motion sensed events on this topic, how about just /snapshot/image?

Reply:
snapshot/image work's I'll update and re-submit